README.md

```diff
@@ -0,0 +1,66 @@
+# Team Awesome 📅
+
+Team Awesome is a "Software as a Service" (SaaS) solution that can be used by Humanoids employees for planning projects, tracking time and much more. On the timesheet page users can record their worked hours a get a clear overview of their work by filtering by client and sorting by date. On the Team members page users can provide information on the projects and clients they worked on. By clicking the green button on the right top-corner users can fill in the forms to provide all the information.
+
+## Installation 💿
+
+To run the Team Awesome app [Node](https://nodejs.org/en/) and [NPM](https://docs.npmjs.com/) are required.
+
+```bash
+# We suggest to use Homebrew as package manager for installing node. In order to install Homebrew we need to pull the latest installation script from the website.
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+# Let's install Node.js using Homebrew:
+Brew install node
+
+# Now you can check your Node version by typing the following command:
+node -v
+```
+
+## Usage 👨🏻‍💻
+
+```bash
+# Run the application in development mode on your localhost
+npm run dev
+
+# To run the app
```

Comment:
These two #'s sound a bit similar. Both run the app, the first one in development mode, but for the second one it's not clear what 'mode this would be'. The second one also has two commands, what's the difference between the two?